### PR TITLE
add re.sonny.Tangram app manifest

### PR DESCRIPTION
**Pull request by sonnyp**
https://github.com/sonnyp/Tangram

### CODE REVIEW

File: re.sonny.Tangram.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "--socket=wayland",
+    "--device=dri",
+    "--share=network",
+    "--socket=pulseaudio",
+    "--filesystem=xdg-data/applications:create"
+  ],
+  "modules": [
+    {
+      "name": "Tangram",
+      "builddir": true,
```
Comment (bilelmoussaoui) [inline re.sonny.Tangram.json]:
```suggestion
```

**Comment by bilelmoussaoui** [comment]
bot, build re.sonny.Tangram

**Comment by bilelmoussaoui** [comment]
The appdata file lacks 
```log
builddir/files/share/appdata/re.sonny.Tangram.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• tag-missing           : <release> required
```

**Comment by sonnyp** [comment]
@bilelmoussaoui done

**Comment by LeandroStanger** [comment]
bot, build re.sonny.Tangram

**Comment by bilelmoussaoui** [comment]
Repository has been created: https://github.com/flathub/re.sonny.Tangram


Thanks!

**Comment by sonnyp** [comment]
@bilelmoussaoui thanks!

Is there an additional step to make it available on https://flathub.org/home ? Otherwise how long will it take?

**Comment by bilelmoussaoui** [comment]
The time for the publish build to finish https://flathub.org/builds/#/builders/29/builds/7407 & to the website cache to be refreshed.
Should be there in a few minutes, maybe an hour or two.

**Pull request closed without merging**